src/surge-fx/SurgeFXEditor.cpp

```diff
@@ -119,8 +119,20 @@ struct Picker : public juce::Component
 
 //==============================================================================
 SurgefxAudioProcessorEditor::SurgefxAudioProcessorEditor(SurgefxAudioProcessor &p)
-    : AudioProcessorEditor(&p), processor(p)
+    : AudioProcessorEditor(&p),
+      sst::jucegui::style::StyleConsumer(sst::jucegui::components::Knob::Styles::styleClass),
+      styleSheet(sst::jucegui::style::StyleSheet::getBuiltInStyleSheet(
+          sst::jucegui::style::StyleSheet::DARK)),
+      processor(p)
 {
+    sst::jucegui::style::StyleSheet::initializeStyleSheets([]() {});
+
+    setTransform(juce::AffineTransform().scaled(1.0));
+
+    // @ BP this compiles but causes a segfault on launc
```

Comment:
@baconpaul this is what I think is supposed to happen but JUCE segfaults on `isShowing()`. I reverted to L153 instead. 

I think once we resolve this issue i can move on to updating the param values.

Reply:
I dug into it more. Its segfaulting here https://github.com/surge-synthesizer/sst-jucegui/blob/main/src/sst/jucegui/style/StyleAndSettingsConsumer.cpp#L49

Reply:
Well that should be in the if but also should not be null 

did you make a non component the consumer?

Reply:
I made the editor a consumer, which is a component. My limited understanding is that the component is null when setting the style which doesn't make sense to me either. 

If I made a non component the consumer, i think it should have failed to compile? 

In my std::cout testing, it failed to even enter the isShowing(). 
I thought it might have been an issue with https://github.com/juce-framework/JUCE/blob/master/modules/juce_gui_basics/components/juce_Component.cpp#L342 But testing showed that it faulted on the entry of `isShowing()` from `sst-jucegui`.